### ISSUE TITLE
The contrast ratio of this colour (used for the place and date) is a bit low and doesn't meet the WCAG accessibility guidelines:

### ISSUE THREAD

**Issue by clytaemnestra** [EuroPython/website#1106]
The contrast ratio of this colour (used for the place and date) is a bit low and doesn't meet the WCAG accessibility guidelines:

<img width="641" alt="image" src="https://github.com/user-attachments/assets/d70fb8ce-d0ce-418d-b4fa-4d292f5df15a" />

<img width="235" alt="image" src="https://github.com/user-attachments/assets/49818ae9-586b-4759-b88d-870ed792e35b" />

The Chrome auto-suggestion is #616392 which is 4.6:

<img width="649" alt="image" src="https://github.com/user-attachments/assets/a55be809-37f1-4078-b614-9213c8e851e9" />

<img width="235" alt="image" src="https://github.com/user-attachments/assets/af73f6d7-9a4b-40cb-b85a-f60c222a288e" />

But it can be something else. Here's a contrast checker: https://webaim.org/resources/contrastchecker/

_Originally posted by @hugovk in https://github.com/EuroPython/website/pull/1050#discussion_r2026824458_

**Comment by nikoshell** [comment]
Fixed.
